src/AbstractCollection.php

```diff
@@ -49,13 +49,13 @@ public function __construct(array $config = [])
     }
 
     /**
-     * Returns true if a collection is empty.
+     * Returns true if the map of identifier objects is empty.
```

Comment:
Please revert. This is an abstract collection for identifiers **and** authenticators.